flyteidl/protos/flyteidl/datacatalog/datacatalog.proto

```diff
@@ -289,6 +289,11 @@ message DatasetID {
     string domain = 3;   // The domain (eg. environment)
     string version = 4;  // Version of the data schema
     string UUID = 5;     // UUID for the dataset (if set the above fields are optional)
+    // Optional, partition key applied to the dataset.
+    string partition = 6;
```

Comment:
can you explain the difference between `partition` and `org` here in the datacatalog?

Reply:
sorry this was find/replace fail - thank you so much for the catch!